.travis.yml

```diff
@@ -113,10 +113,10 @@ script:
 
 matrix:
   include:
-    - php: 5.6
-      env: "DB=mysql CORE_BRANCH=stable12 TEST_JS=TRUE"
     - php: 7.0
-      env: "DB=mysql CORE_BRANCH=stable12 PACKAGE=TRUE"
+      env: "DB=mysql CORE_BRANCH=master TEST_JS=TRUE"
+    - php: 7.1
+      env: "DB=mysql CORE_BRANCH=master PACKAGE=TRUE"
```

Comment:
Please also add tests against `CORE_BRANCH=stable13` if you still support that :wink: 

Also please add php7.2 tests

Reply:
Please see the full file. Stable13 had already been the default core branch and we test php56 to php72 :wink:

Reply:
Ah my bad. Man I hate this weird travis syntax :P